### PR TITLE
Add printing SOP Instance UID to test method GetSopInstanceUID.

### PR DESCRIPTION
**Pull request by asamitov**
Add missing test of GetSopInstanceUID in SandboxTest.

### CODE REVIEW

File: SandboxTest/Main.cpp [hunk 1]
```diff
@@ -227,6 +227,12 @@ int wmain (int argc, wchar_t *argv[]) {
         CHECK(loader->GetImageSource(&source));
     }
 
+    {
+        CComBSTR sopInstanceUID;
+        CHECK(source->GetSopInstanceUID(&sopInstanceUID));
+        std::wcout << "SOP Instance UID: " << sopInstanceUID.m_str << L"\n";
```
Comment (forderud) [inline SandboxTest/Main.cpp]: One small comment: Can you convert the "SOP Instance UID: " text string to wide-char, so that it matches "wcout"?

Comment (asamitov) [reply SandboxTest/Main.cpp]: Good catch.

Comment (asamitov) [reply SandboxTest/Main.cpp]: Done. Also fixed missed `L` near `"Probe name"`

**Comment by asamitov** [comment]
@forderud Console output looks correct. Unfortunately, I have no write access to merge.

**Pull request merged**